src/Main.java

```diff
@@ -1,5 +1,99 @@
+System.out.println("Значение переменной <b> с типом данных short равно " +b);
+System.out.println("Значение переменной <c> с типом данных int равно " +c);
+System.out.println("Значение переменной <d> с типом данных long равно " +d);
+System.out.println("Значение переменной <e> с типом данных float равно " +e);
+
+        System.out.println("Задание 2");
+        float x = 27.12f;
+        float klass = 2.786f;
+        long glass = 987_678_965_549L;
+        short arrow = 569;
+        short frog = -159;
+        short home = 27897;
+        byte gnom = 67;
+
+        System.out.println("Задание 3");
+byte class1 = 23;
+byte class2 = 27;
+byte class3 = 30;
+short totalLeaf = 480;
+short totalStudent = (short) (class1 + class2 + class3);
+short totalLeafStudent = (short) (totalLeaf / totalStudent);
+System.out.println("На каждого ученика расчитано " +totalLeafStudent+ " листов");
+
+        System.out.println("Задание 4");
+        byte productionProductivity = 16 / 2; //в минуту
+        short production20min = (short) (productionProductivity * 20);
+        short production1day =(short) (productionProductivity * 60 * 24);
+        int production3day = production1day * 3;
+        int production1ear = production1day * 365;
+System.out.println("За 20 минут машина произвела " +production20min+ " штук бутылок");
+System.out.println("За 1 день машина произвела " +production1day+ " штук бутылок");
+System.out.println("За 3 дня машина произвела " +production3day+ " штук бутылок");
+System.out.println("За год машина произвела " +production1ear+ " штук бутылок");
+
+        System.out.println("Задание 5");
+        byte whitePot = 2;
+        byte brownPot = 4;
+        byte potPaint =(byte) (whitePot + brownPot); // банок краски на класс
+        byte totalPot = 120;
+        byte totalklass =(byte) (totalPot / potPaint);
+        byte totalWhitePot =(byte) (totalklass * whitePot);
+        byte totalBrownPot =(byte) (totalklass * brownPot);
+System.out.println("В школе, где " +totalklass+ " классов,нужно " +totalWhitePot+ " банок белой краски и " +totalBrownPot+ " банок коричневой краски");
+
+        System.out.println("Задание 6");
+        short bananWeight = 5 * 80;
+        float milkWeight = 0.971f * 200; //плотность молока
+        short plombirWeight = 2 * 100;
+        short eggWeight = 4 * 70;
+        float dishWeightTotal = bananWeight +  milkWeight + plombirWeight + eggWeight;
+        float dishWeightTotalKG = ((bananWeight + milkWeight + plombirWeight + eggWeight)/1000);
+        double dishWeightTotalKG1 = Math.round(dishWeightTotalKG * 1000.0) / 1000.0;
+        System.out.println( +dishWeightTotal+ " грамм готового продукта получается, либо " +dishWeightTotalKG1+ " в килограммах");
+
+            System.out.println("Задание 7");
+
+            byte throwOffWeight = 7;
+            short lostWeight250 = (7 * 1000) / 250;
+            short lostWeight500 = (7 * 1000) / 500;
```

Comment:
переменную надо использовать
```suggestion
            short lostWeight250 = (throwOffWeight * 1000) / 250;
            short lostWeight500 = (throwOffWeight * 1000) / 500;
```